config/go-app-ussd_public.cgg_UG.json

```diff
@@ -0,0 +1,402 @@
+        null,
+        "Bambe taasamu enamba yaabo yomuhagatiro. Ekyokureberaho 0803304899"
+    ],
+    "We did not recognise the VHTs number.": [
+        null,
+        "Titwetegyerize enamba yomushaho waawe wahakyaro."
+    ],
+    "What is the name of your parish?": [
+        null,
+        "Omuruka gwawe niguuha?"
+    ],
+    "Select your parish": [
+        null,
+        "Torana omuruka"
+    ],
+    "Sorry, there are no results for your parish name.": [
+        null,
+        "Bambe, tihaine birikwooreka ahabyomuruka gwaawe."
+    ],
+    "Please re-enter your parish name carefully and make sure you use the correct spelling.": [
+        null,
+        "Bambe garuka otaasemu eizina ryomuruka kurungi kandi ogyezeho kurihandika gye"
+    ],
+    "Thank you. Your FamilyConnect ID is {{health_id}}. You will receive an SMS with it shortly.": [
+        null,
+        "Yebare. Enambaya FamilyConnect ni {{health_id}}. Nooza kutunga obutumwa juba juba."
+    ],
+    "Thank you for using the FamilyConnect service": [
+        null,
+        "Webaare kukwozesa obuhereeza oburikugeita/ oburikushuganisa eka"
+    ],
+    "Sorry, invalid option.": [
+        null,
+        "Bambi watorana kubi."
+    ],
+    "Sorry, invalid number.": [
+        null,
+        "Enamba egweire."
+    ],
+    "Sorry, invalid date.": [
+        null,
+        "Bambe Ebiro bigweire."
+    ],
+    "Sorry, invalid name.": [
+        null,
+        "Bambe Enamba egweire."
+    ],
+    "Yes": [
+        null,
+        "Eego"
+    ],
+    "No, start from the beginning": [
+        null,
+        "Eingaaha, tandika kuhandika busya"
+    ],
+    "English": [
+        null,
+        "Orujungu"
+    ],
+    "Rukiga": [
+        null,
+        "Orukiga"
+    ],
+    "Register this number to receive SMSs about you & your baby": [
+        null,
+        "Handikisa enamba yo kuherezaho obutumwa bwo mwana wawe"
+    ],
+    "Manage a different phone number": [
+        null,
+        "Kozesa enamba endijo"
+    ],
+    "Start baby SMSs": [
+        null,
+        "Tandika obutumwa bwomwaana"
+    ],
+    "Update language": [
+        null,
+        "Tereza ita ahamutindo orurimi"
+    ],
+    "Change the number which gets SMSs": [
+        null,
+        "Hindura enamba erikutunga obutumwa"
+    ],
+    "Stop SMSs": [
+        null,
+        "Rekyeeraho obutumwa"
+    ],
+    "Back to main menu": [
+        null,
+        "Garuka ahakubanza"
+    ],
+    "Exit": [
+        null,
+        "Rugaho"
+    ],
+    "Try a different number": [
+        null,
+        "Gyezaho enamba endijo"
+    ],
+    "Head of the Household": [
+        null,
+        "Nyineka"
+    ],
+    "Mother to be": [
+        null,
+        "Omukazi orikuza kuzara"
+    ],
+    "Family member": [
+        null,
+        "Omuntu ondijo owomuka omu"
+    ],
+    "Trusted friend": [
+        null,
+        "Munywani wawe omwesigwa"
+    ],
+    "Mother miscarried": [
```

Comment:
This hasn't been updated along with the copy here: https://github.com/praekelt/chms-jsbox/pull/56/files#diff-f11efc6ebde289613436f88de0401d97L1536